client/js/screens/Login/Login.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import {
+  TouchableOpacity,
+  AsyncStorage,
+  View,
+  Text,
+  Image,
+  TextInput
+} from "react-native";
+import { Form, Field } from "react-final-form";
+import styles from "./styles";
+
+class Login extends Component {
+  static navigationOptions = {
+    title: "Login"
+  };
+
+  handleInput = () => {
+    console.log("something submitted");
+  };
+
+  goToRegister = () => {
+    this.props.navigation.navigate("Register");
+  };
+
+  _signInAsync = async () => {
+    await AsyncStorage.setItem("userToken", "abc");
```

Comment:
create a model.js file where you can store all your DB related methods and call them from anywhere in your app by importing them.